lib/pages/home/home_page.dart

```diff
@@ -10,23 +10,25 @@ import 'package:survey/pages/home/survey_carousel/survey_carousel_card.dart';
 class HomePage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    RefreshController _refreshController =
+    RefreshController pullToRefreshController =
         RefreshController(initialRefresh: false);
+
+    PageController pagingController = PageController();
```

Comment:
I'm afraid that we still need `dispose` for this `PageController` 🤕 https://api.flutter.dev/flutter/widgets/PageController-class.html

Reply:
@manh-t this page is a StatelessWidget and it doesn't have `dispose()` 😅  as the widget doesn't mean to have a new state once it's created so disposing the state isn't required imo

Reply:
@sleepylee `as the widget doesn't mean to have a new state once it's created so disposing the state isn't required`, I agree with you somehow on this point. 🤸‍♂️